flask/app.py

```diff
@@ -1370,6 +1392,32 @@ def url_defaults(self, f):
         self.url_default_functions.setdefault(None, []).append(f)
         return f
 
+    def _find_error_handler(self, e):
+        """Finds a registered error handler for the request’s blueprint.
+        If neither blueprint nor App has a suitable handler registered, returns None
+        """
+        exc_class, code = self._ensure_exc_class(e)
+        
+        def find_superclass(d):
+            if not d:
+                return None
+            for superclass in exc_class.mro():
+                if superclass is BaseException:
+                    return None
+                handler = d.get(superclass)
+                if handler is not None:
+                    return handler
```

Comment:
>You mean exc_class

Ah, yes, sorry.

>also during runtime (after that), some new error handler is registered. (not inconceivable)

This is explicitly forbidden by Flask. See the ``setupmethod`` decorator.